### DISCUSSION TITLE
How to stick with one animation

### DISCUSSION THREAD

**Discussion by popy2k14** [kageurufu/steamdeck_startup_animations#17] [Q&A] [answered]
Hey guys.

Thanks fo this amazing addon for our beloved deck.
How can i stick with one animation?
Is it just as simple as disable the random service?

thx

**Comment by drashna** [comment, marked as answer]
Yup. `systemctl --user disable randomize_deck_startup.service`

**Comment by popy2k14** [comment]
Thx, marked as answer
